tests/wdio.conf.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * Ghostery Browser Extension
+ * https://www.ghostery.com/
+ *
+ * Copyright 2017-present Ghostery GmbH. All rights reserved.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0
+ */
+
+import path from 'node:path';
+import url from 'node:url';
+import { promises as fs } from 'node:fs';
+
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
+const extPath = path.join(__dirname, '..', 'web-ext-artifacts');
+
+export const config = process.env.DEBUG
+  ? {
+      specs: [['*.spec.js']],
+      capabilities: [
+        {
+          browserName: 'chrome',
+          'goog:chromeOptions': {
+            args: [`--load-extension=${path.join(__dirname, '..', 'dist')}`],
+          },
+        },
+      ],
+      injectGlobals: false,
+      reporters: ['spec'],
+      logLevel: 'warn',
+      mochaOpts: { timeout: 24 * 60 * 60 * 1000 },
+    }
+  : {
+      specs: ['*.spec.js'],
+      capabilities: [
+        {
+          browserName: 'chrome',
+          'goog:chromeOptions': {
+            args: [
+              'headless',
+              'disable-gpu',
+              `--load-extension=${path.join(extPath, 'ghostery-chromium')}`,
+            ],
+          },
+        },
+        {
+          browserName: 'firefox',
+          'moz:firefoxOptions': { args: ['-headless'] },
+        },
+      ],
+      injectGlobals: false,
+      reporters: ['spec'],
+      logLevel: 'silent',
+      mochaOpts: { timeout: 24 * 60 * 60 * 1000 },
+      before: async (capabilities, specs, browser) => {
+        if (capabilities.browserName === 'firefox') {
+          const extension = await fs.readFile(
+            path.join(extPath, 'ghostery-firefox.zip'),
```

Comment:
both browsers are able to run unpacked addons. maybe webdriverio does have a first class support for that, but it is not hard to achieve.

Reply:
If you mean loading not exported internal part of the `web-ext` library for Firefox, I would keep it as it is:
https://github.com/ghostery/webextension-event-recorder/blob/main/src/firefox.js#L3

I would even prefer to load zips in both browsers, but Chrome does not support it (as far as I have checked all the arguments and documentation). 

And after all, what's wrong with loading the zip? The WDIO driver supports that out of the box, we want to test the production version, and we have to make both builds parallel - so still I would have to copy the `dist/` somewhere.

Reply:
Unpacking the extension increases the duration of the test, which is not some big problem, as well as generating a few lines of extra code to maintain, which is also not a problem. Nevertheless, in my opinion, it is always safer to unzip first - it can be considered as a kind of safeguard that the extension is downloaded at all.

Reply:
Chrome loads unpacked extensions just fine https://github.com/ghostery/webextension-event-recorder/blob/e41cd33260f48128ef5e195b35f2b607316b62e6/src/chrome.js#L12

Reply:
I think @GRadziejewski misunderstood that - it is just the opposite, but the extension is loaded only once per session, for all tests... In Chrome we load the unpacked version (there is no way to load zip), and in Firefox we load zip (as in my opinion loading the unpacked version does not give any advantage, and adds a compilation to the implementation). After all, it has no impact on running tests.

Reply:
when it comes to `web-ext` we can make a PR to project to export the `remote`